samples/nrf9160/coap_client/README.rst

```diff
@@ -18,6 +30,7 @@ Requirements
   * |nRF9160DK|
 
 * .. include:: /includes/spm.txt
+* A public CoAP server IP address available on the internet
```

Comment:
Should we be adding this as a requirement ?

Reply:
I think it's fine, it does not the be an IP address though, you can use URL.

Reply:
Modified it as :  A public CoAP server IP address or URL available on the internet